scratchapi2/gclass.py

```diff
@@ -1,13 +1,17 @@
-""" GClass is a functional class library. """

```

Comment:
"a functional class library" sounds like exaggeration. Docstrings aren't for advertising, they're for documenting.